nn.js

```diff
@@ -44,8 +44,8 @@ function mutate(x) {
 }
 
 // Neural Network constructor function
-function NeuralNetwork(inputnodes, hiddennodes, outputnodes, activation) {
-
+function NeuralNetwork(inputnodes, hiddennodes, outputnodes, learningRate) {
```

Comment:
shouldn't this be `function NeuralNetwork(inputnodes, hiddennodes, outputnodes, learningRate, activation) {`?